%YAML:1.0
---
camera_matrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 7.7056641957454394e+02, 0., 3.0880885456079449e+02, 0.,
       7.6784399215121357e+02, 2.6685078503437217e+02, 0., 0., 1. ]
dist_coeff: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ 2.0690272870148838e-01, -5.7440771219654996e-01,
       -8.5614374042434233e-03, -2.0844663845359492e-03,
       -2.5337310947904568e+00 ]
